packages/Frontend/src/components/NewJournal.jsx

```diff
@@ -1,5 +1,68 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { Button } from 'semantic-ui-react';
+import ReactQuill from 'react-quill';
+import styled from 'styled-components';
+import 'react-quill/dist/quill.snow.css';
 
-export default function NewJournal() {
-  return (<h3>Create New Journal</h3>);
+const StyledDiv = styled.div`
+  height: 90%;
+  display: block;
+  position: relative;
+  .quill {
+    height: 90%;
+  }
+`;
+
+export default class NewJournal extends Component {
+  static modules = {
+    toolbar: [
+      [{ header: [1, 2, false] }],
+      ['bold', 'italic', 'underline', 'strike'],
+      [{ color: [] }, { background: [] }],
+      [{ align: [] }, { list: 'ordered' }, { list: 'bullet' }],
+      ['link', 'image', 'video'],
+    ],
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = { text: '' };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(value) {
+    this.setState({ text: value });
+  }
+
+  render() {
+    return (
+      <div style={{
+          marginLeft: 'auto',
+          marginRight: 'auto',
+          position: 'relative',
+          top: 40,
+          width: '70%',
+          height: '50em',
+        }}
+      >
+        <StyledDiv>
+          <ReactQuill
+            modules={NewJournal.modules}
+            onChange={this.handleChange}
+            value={this.state.text}
+          />
+        </StyledDiv>
+        <div style={{
+          display: 'block',
+          float: 'right',
+          position: 'relative',
+          top: '1em',
+        }}
+        >
+          <Button color="blue">Save</Button>
+          <Button color="violet">Save & Publish</Button>
```

Comment:
If we want to automatically save things like Google Drive does, then we should drop the save button and change save & publish to just publish. Where the save button is we can have a small p tag that has a loading wheel while it's being saved and then uses moment to say something like "Saved 2 minutes ago", what do you think? I can help you detail exactly what it would take to do all of that, but I just need to know if we want to go that route.